gateway/modules/eventing/eventing.go

```diff
@@ -121,3 +126,36 @@ func (m *Module) SetConfig(project string, eventing *config.Eventing) error {
 
 	return nil
 }
+
+// CloseConfig closes the module config
+func (m *Module) CloseConfig() error {
+	//erase map
+	m.eventChanMap.Range(func(key interface{}, value interface{}) bool {
```

Comment:
@sharadregoti I think each channel gets closed eventually right? So we don't need to do this I guess